cleanlab/datalab/internal/issue_manager/label.py

```diff
@@ -86,11 +86,11 @@ def __init__(
     def _process_find_label_issues_kwargs(**kwargs: Dict[str, Any]) -> Dict[str, Any]:
         """Searches for keyword arguments that are meant for the
         CleanLearning.find_label_issues method call
-
+    
         Examples
         --------
         >>> from cleanlab.datalab.internal.issue_manager.label import LabelIssueManager
-        >>> LabelIssueManager._process_find_label_issues_kwargs(thresholds=[0.1, 0.9])
+        >>> LabelIssueManager._process_find_label_issues_kwargs(**{'thresholds': [0.1, 0.9]})
```

Comment:
Revert this to the previous example:

```suggestion
        >>> LabelIssueManager._process_find_label_issues_kwargs(thresholds=[0.1, 0.9])
```